stl_text/ops/tokenizers/whitespace_tokenizer.py

```diff
@@ -1,44 +1,57 @@
+import torch
 import torch.nn as nn
 from typing import Dict, List, Optional
+from collections import OrderedDict
 from iopath.common.file_io import PathManager
-
+from torchtext.experimental.vocab import (
+    load_vocab_from_file,
+    Vocab,
+)
+from torchtext._torchtext import Vocab as VocabPybind 
 
 class WhitespaceTokenizer(nn.Module):
-    def __init__(self, vocab_path: Optional[str] = None, trainable=False, speed: int = 0):
+    """
```

Comment:
you can follow the doc string format [here](https://github.com/pytorch/text/blob/master/torchtext/experimental/transforms.py#L77).

Reply:
> you can follow the doc string format [here](https://github.com/pytorch/text/blob/master/torchtext/experimental/transforms.py#L77).

sure, thanks for the pointer. Do we still want to keep JIT output in the doc string?